vignettes/hys_law.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Hy's Law Implementation Guide"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Hy's Law Implementation Guide}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiral)
+library(dplyr, warn.conflicts = FALSE)
+library(tidyr, warn.conflicts = FALSE)
+library(purrr)
+library(admiral.test)
+library(lubridate)
+library(stringr)
+```
+
+# Context
+During the drug development process, clinical trials are often required to assess for the potential that the experimental drug can cause severe liver injury, known as a drug induced liver injury (DILI). There are multiple criteria that need to be evaluated to determine and classify a DILI "event". A common rule of thumb for this is, known as Hy's Law, is usually comprised of three parts:
+
+- Elevated alanine transaminase (ALT) or aspartate transaminase (AST) by 3-fold or greater of the upper limit of normal, or 5-fold (in the case of aminotransferases instead of transaminases).
+- Elevated serum total bilirubin (BILI) by 2-fold or greater within a window of time, ~14 days after the elevated AST/ALT event.
+- No other reason to explain these increased lab values like preexisting liver disease.
+
+
+# Example
+We can use a mixture of `admiral` and basic `tidyverse` functions to assist us in creating an ADLBHY dataset. In the walkthrough below we will use the adlb dataset created from the call `use_ad_template("adlb")`
+
+```{r}
+adlb <- admiral_adlb
+head(adlb)
+```
+
```

Comment:
can we use the dataset_vignette function in admiraldev to display the data?

Reply:
Noted!